charts/pgadmin4/templates/_helpers.tpl

```diff
@@ -31,6 +31,16 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+tpl to a map a map/string that is provided by a toMap key and context. 
+Usage:
+{{- include "pgadmin.tplToMap" (dict "toMap" .Values.myExampleMap "context" $) }}
+{{- include "pgadmin.tplToMap" (dict "toMap" .Values.myExampleString "context" $) }}
```

Comment:
What is your thought on leaving this as a comment? The Usage is not really needed, since this is not a common helm chart.

```suggestion
Render a value as a map after templating it.
Supports both raw string and map types as input.
```